domain/src/test/java/org/oppia/android/domain/question/QuestionAssessmentProgressControllerTest.kt

```diff
@@ -1286,6 +1286,214 @@ class QuestionAssessmentProgressControllerTest {
       .isEqualTo(skill1Score)
   }
 
+  @Test
+  fun solutionViewedForAllQuestions_returnMaxMasteryLossPerQuestion() {
+    setUpTestApplicationWithSeed(questionSeed = 0)
+    subscribeToCurrentQuestionToAllowSessionToLoad()
+    // This will generate question 1 (skill 0), question 2 (skill 0), and question 3 (skill 1)
+    startTrainingSession(TEST_SKILL_ID_LIST_01)
+
+    // Question 1
```

Comment:
Can you add a comment below that you are submitting a wrong answer here too?

Reply:
Done!